src/Post.vue

```diff
@@ -78,29 +84,29 @@ export default {
           vm.error = this.statusText;
         } else {
           let parser = new DOMParser();
-          let doc = parser.parseFromString(self.responseText, 'text/html')
+          let doc = parser.parseFromString(self.responseText, 'text/html');
           let metadata = doc.querySelector('table');
-          metadata.parentNode.removeChild(metadata)
-          vm.parseMetaData(metadata)
+          metadata.parentNode.removeChild(metadata);
+          vm.parseMetaData(metadata);
           vm.post = doc.querySelector('#file').outerHTML;
           vm.loading = false;
         }
-      }
+      };
       xhr.send();
     },
     // Since this is a really fast operation and we don't want our user to hang/wait for the title of the post, synchronous request is o.k.
     parseMetaData(table) {
       let tbody = table.querySelectorAll('tbody tr td');
-      table.querySelectorAll('thead tr th').forEach((item) => {
```

Comment:
Is there no way to use a vue component instead in place of this table? This way you could just fire a global event that you can listen to in the other component and then do the replacement in vue template. then there is no html updating mechanism required here.